Installation + Usage suggestions on README

whitecl opened the pull request.

Commit:
Usage section on README

Commit:
Add installation notes

randycoulman commented on README.md:
```diff
@@ -4,6 +4,52 @@
 # Cellulose
 
-Cellulose is a react layout library based on contextually aware components using flexbox.
+Cellulose is a React layout library based on contextually aware components using flexbox.
```
Maybe "based on" -> "that allows you to create"

randycoulman commented on README.md:
````diff
@@ -4,6 +4,52 @@
+)
+```
+
+With Cellulose, changing styles responsively based on breakpoints is simple
````
It's a good idea to avoid terms like "simple", "easy", or "basic" so that people don't feel stupid if they don't find it simple.

So maybe something like, "The spanOptions can provide a number of columns for each breakpoint, or they can provide an object containing the number of columns as well as any other breakpoint-specific properties such as `className`."

randycoulman reviewed (changes requested):
I assume richer documentation is coming, but I feel like there's a motivating paragraph missing at the beginning.  Maybe it's because "contextually-aware components" doesn't mean anything to me.

Maybe something along these lines:

"Traditionally, responsive web apps change their layout based on the size of the browser window by specifying "breakpoints" - window widths at which the layout changes.

Cellulose allows each component to control its layout based on the width of its containing component instead of the browser window.  This allows components to be re-used in a variety of contexts and to be responsive based on the width of each context, rather than responsive only to the entire browser window width."

Not perfect, but you get the idea, I hope.

Commit:
More README

Better summary of what Cellulose does.
Better examples and explanations of how to get basic benefits + how to
use responsive grid

Commit:
Flesh out examples as full React components

randycoulman commented on README.md:
````diff
@@ -4,52 +4,92 @@
 ```
 
-With Cellulose, changing styles responsively based on breakpoints is simple
+Cellulose can also be used to create a responsive grid. Use the prop `columns=` of `<Cellulose>` to define a quantity of columns, then add `<Cell>` components with `spanOptions` props that define responsive behavior
````
"quantity of columns" feels a bit awkward.  Maybe "number of columns to use"?

randycoulman commented on README.md:
```diff
@@ -4,52 +4,92 @@
-    </Cell>
-  </Cellulose>
-)
+Cellulose's `<Cell>` components can even configure their own classes based on which `breakPoints` are activated!
```
Is `className` the only prop you can configure in addition to `cols`?  If not, we might want to mention the others here.

whitecl replied:
It looks like there are more options to Cell. Perhaps a table of props after Usage would be a good way to detail this...

randycoulman reviewed (approved):
This looks great!  Nice improvements!

I made a couple of optional suggestions that you can take or leave.

Commit:
Wording change

Thanks @randycoulman!
